app/ide-desktop/lib/dashboard/src/components/SvgMask.tsx

```diff
@@ -20,19 +20,16 @@ export interface SvgMaskProps {
   // underlying `div`.
   // eslint-disable-next-line no-restricted-syntax
   readonly className?: string | undefined
-  readonly onClick?: (event: React.MouseEvent) => void
 }
 
 /** Use an SVG as a mask. This lets the SVG use the text color (`currentColor`). */
 export default function SvgMask(props: SvgMaskProps) {
-  const { invert = false, alt, src, title, style, color, className, onClick } = props
+  const { invert = false, alt, src, style, color, className = 'h-max w-max' } = props
```

Comment:
oh, maybe it's better to merge custom styles(from className with the default one), using twMerge?